### PR TITLE
Fixes #7367: Add compliance API documentation

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/7367

### CODE REVIEW

File: src/api_v2/compliance.md [hunk 1]
```diff
@@ -0,0 +1,449 @@
+
+This file describe REST API v2 for Rules.
+
+Corresponding code is available here: 
+https://github.com/Normation/rudder/blob/master/rudder-web/src/main/scala/com/normation/rudder/web/rest/rule/RuleAPI2.scala
+
```
Comment (fanf) [inline src/api_v2/compliance.md]: I'm not sur that you wanted to put that introduction for compliance, did you ?

File: src/api_v2/compliance.md [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+        "result": "success",
+        "action": "getNodesCompliance"
+      }
+
+
+    */
+
+
+[GET] api/compliance/nodes/rules/{id}
+-----------------
```
Comment (fanf) [inline src/api_v2/compliance.md]: It's api/compliance/nodes/{id} no ?

**Comment by VinceMacBuche** [comment]
PR updated

File: src/api_v2/compliance.md [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+
+
+    */
+
+
+[GET] api/compliance/nodes/{id}
+-----------------
+
+    /**
+    @api {get} api/compliance/nodes/rules/{id} 4. Get Compliance details of a Node
```
Comment (fanf) [inline src/api_v2/compliance.md]: still not correct here

**Comment by VinceMacBuche** [comment]
PR updated

**Pull request merged**